src/Draco.Compiler/Api/Syntax/TokenKind.cs

```diff
@@ -65,6 +65,11 @@ public enum TokenKind
     /// </summary>
     StringNewline,
 
+    /// <summary>
+    /// An escape sequence in a string.
+    /// </summary>
+    EscapeSequence,
```

Comment:
Guess who need to update their formatter

Reply:
MUHAHAHAHAHAHA